bscript/interpreter/engine_test.go

```diff
@@ -957,3 +957,32 @@ func TestExecute(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestExecuteUnknownBug(t *testing.T) {
```

Comment:
This is not a good name for a test function. IMO it would be better to move the test inside `TestExecute(t *testing.T)` as the code is mostly the same. Ideally you transform `TestExecute(t *testing.T)` into a table based test. In this way the test of the Execute function is more readable and the test file is less bloated.

I created a branch with an additional commit here to show what I mean: https://github.com/libsv/go-bt/commit/29cf7fb51c37a687b2951e6a4924df8e616d75d8

Reply:
Ideal thanks, please merge the table version you've made, much cleaner.